micro-rdk/src/common/motor.rs

```diff
@@ -106,6 +106,13 @@ pub trait Motor: Status + Actuator + DoCommand {
     /// If revolutions is 0, this will run the motor at rpm indefinitely.
     /// If revolutions != 0, this will block until the number of revolutions has been completed or another operation comes in.
     fn go_for(&mut self, rpm: f64, revolutions: f64) -> Result<Option<Duration>, MotorError>;
+    // Instructs the motor to turn at the specified RPM. The default behavior is to call go_for
```

Comment:
Personal preference nit, feel free to ignore: could you throw blank lines in this trait declaration before starting a new comment block?